cmd/kola/kola.go

```diff
@@ -128,6 +128,7 @@ func writeProps() error {
 	type AWS struct {
 		Region       string `json:"region"`
 		AMI          string `json:"ami"`
+		ResolvedAMI  string `json:"ami-id",omitempty`
```

Comment:
I suspect consumers won't care how the original request was phrased, and won't want to check two different fields for the AMI ID.  I'd suggest putting the ID in `AMI` in all cases.